### DISCUSSION TITLE
Number of WIP is different between Daily Work in Progress and Daily WIP x Work Item Age charts

### DISCUSSION THREAD

**Discussion by bsantosbhz** [nbrown02/FlowViz#143] [General]
I'm trying to figure out why the number of work items is different when comparing the "Daily Work in Progress" and "Daily WIP x Work Item Age" charts. Additionally, the value for a single day in the "Daily Work in Progress" chart does not match the Y-axis value. In the second image, the number of items with an age greater than 14 days is higher than the 20 work items shown in the first image.

![image](https://github.com/user-attachments/assets/7746c53a-dc0f-43e7-b489-990fed1ab36e)

![image](https://github.com/user-attachments/assets/9d04fea6-ac3c-4c10-a00b-bbff18ca4088)

**Comment by nbrown02** [comment]
Hi Bruno,

Thanks for sharing this - it looks like a bug that needs fixing. I will take a look at this tomorrow 👍🏼

**Comment by nbrown02** [comment]
Hi Bruno - I've taken a look. The issue is that the tooltip and chart are using different tables. The chart is using just a count of 'WIP', which may mean items that are not mapped to a board column at user story level. It seems to me that you have lots of 'hidden WIP' - I'd look at the WorkItems WIP Age table to see if all those items are at user story/PBI level.

**Comment by bsantosbhz** [comment]
Thanks for replying, Nico. I'll take a look based on your comments.

Em qua., 6 de nov. de 2024 às 04:58, Nick Brown ***@***.***>
escreveu:

**Reply by ImEdmond** [reply]
@nbrown02 Yes. I´m having the same problem. Daily WIP x WorkAge is correct (all statuses - To Do and Done) but Daily Work in Progress has 34 plus PBIs
![image](https://github.com/user-attachments/assets/32525ae8-e249-4c65-9e61-508123ac4881)

**Reply by nbrown02** [reply]
Both those charts show the same value and are from the same table?

**Reply by ImEdmond** [reply]
@nbrown02 my bad. You can close this one as stupidity of user. I´m double checking the number of the tool against JIRA. I was considering only the open sprint thus the difference. Daily Work in Progress and Daily WIP x Work Item Age are matching like a charm. Off topic:  I was looking at the tables and it does not seem that there is an obvious way to get sprint information only.
